### ISSUE TITLE
Revise the use of Organization.alias as Acronym

### ISSUE THREAD

**Issue by alexanderkiel** [samply/bbmri-fhir-ig#76]
We use Organization.alias in the Biobank and the Collection profile as a mapping for "acronym" from the Directory and MIABIS. 

In FHIR the definition of Organization.alias is:

>A list of alternate names that the organization is known as, or was known as in the past.

As it may be possible to count an acronym as an alternative name, we rely on restricting the cardinality to a maximum of one in order to have at most one alias to choose from and map to acronym. Restricting the alias on such a way hinders interoperability because biobanks/collections no longer can have multiple alternative names.

I would suggest to use an identifier for the acronym.

**Comment by noemide** [comment]
Using identifier sounds good, but i see a few possible issues. What schould be the system URL? How do we ensure uniqueness (required by the identifier type)? Maybe we could look to the BBMRI-Directory, if they ensure the uniqueness.

**Comment by alexanderkiel** [comment]
> Maybe we could look to the BBMRI-Directory, if they ensure the uniqueness.

Yes, you are right, we should clarify that first.

**Comment by noemide** [comment]
Ok, so "Alias"-values are arbitrary text fields in the Directory. No strict uniqueness / rules applied. Should we try tp think of another approch or close this issue?

**Comment by alexanderkiel** [comment]
I would suggest to use an extension here. We should not constrain the number of aliases to one and also alias is something different as an acronym. What do you think?

**Comment by noemide** [comment]
I agree with using an extension. Since it would require updating examples, we should first migrate all profiles and extensions connected to biobank and collection to FSH.
